docs/installation-guide-azure.md

```diff
@@ -198,15 +198,22 @@ cd build
 ```
 
 ```
+export NAME=cluster1
 for i in envs.sh backend.tf; do
   az storage blob download --account-name ${NAME}terraform -c ${NAME}-build -n ${i} -f ${i}
 done
 ```
 
+Command below will ask for secrets that can be found in keepass.
+
+```
+source envs.sh
```

Comment:
So, is that one for all, or creds per account?

Reply:
per account